### PR TITLE
Playlists

### PR DESCRIPTION
**Pull request by JantuDeb**
#Added Playlists
- create playlist
- remove playlist
- add a video to the playlist
- remove videos from the playlist

image
image

### CODE REVIEW

File: src/components/player/Player.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.video-wrapper{
+  width: 60%;
+  flex-grow: 1;
+}
+
+/* .react-player {
+  width: 100%;
+  height: fit-content;
+} */
+
```
Comment (mariyasada) [inline src/components/player/Player.css:10]: remove this if not necesary

File: src/context/videos/VideoContext.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const [videos, setVideos] = useState([]);
+  const [categories, setCategories] = useState([]);
+  const [selectedCategory, setSelectedCategory] = useState("");
+  useEffect(() => {
+    (async () => {
+      const [videoResponse, categoryResponse] = await Promise.all([
+        axiosInstance.get("/videos"),
+        axiosInstance.get("/categories"),
+      ]);
+      setVideos(videoResponse.data.videos);
```
Comment (mariyasada) [inline src/context/videos/VideoContext.jsx:15]: liked you have used promise.all

**Review by mariyasada — APPROVED** [review]
I didn't get the axioxprivate but the code structure and UI is good

**Comment by JantuDeb** [comment]
> I didn't get the axioxprivate but the code structure and UI is good

Instance of `axios` which will add authentication token for all request

**Review by ajit-sharmaGH — APPROVED** [review]
Everything looks fine 
Great logic 
You can merge it

**Pull request merged**